server/conf/timely-standalone.yml

```diff
@@ -0,0 +1,47 @@
+timely:
+  ip: 0.0.0.0
+  port:
+    put: 54321
+    query: 54322
+    websocket: 54323
+  instance-name: TimelyStandalone
```

Comment:
Agreed it might be a little clunky. However, if I move http-port into http and websocket-port into websocket, then where does tcp-port go? Also, where would ip go, since it's kind of associated with each port (though there's a bug where it isn't actually used right now).